backend/src/Service/Admin/Captain/AdminCaptainService.php

```diff
@@ -42,6 +46,11 @@ public function getCaptainProfileByIdForAdmin(int $captainProfileId): ?CaptainPr
             if($captainProfile['roomId']) {
                 $captainProfile['roomId'] = $captainProfile['roomId']->toBase32();
             }
+            
+            $captainProfile['images'] = $this->uploadFileHelperService->getImageParams($captainProfile['images']);
```

Comment:
You can create a class  and use it for captain images:

`class CapatinImagesHelper`

so you do not repeat this lines